Makefile

```diff
@@ -26,11 +26,11 @@ vendor: go.mod go.sum
 .PHONY: lint-prepare
 lint-prepare:
 	@echo "Installing golangci-lint"
-	@GO111MODULE=off go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+	@wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.31.0
```

Comment:
why not using go get?

Reply:
because this is the recommended way from `golangci-lint`. based on their documentation `go get` installation aren't guaranteed to work.